source/tokens/Token.hx

```diff
@@ -0,0 +1,21 @@
+package tokens;
+
+import flixel.FlxBasic;
+import flixel.util.FlxColor;
+import obstacle.Obstacle;
+
+class Token extends Obstacle
+{
+	override function init(parameters:ObstacleParameters)
+	{
+		super.init(parameters);
+		color = FlxColor.YELLOW;
+	}
+
+	public static function overlapsWithPlayer(player:FlxBasic, scoreToken:Token)
+	{
+		scoreToken.kill();  //Remove token when collide with player
+		//scoreToken.score += 1;    //Add 1 to coin score when player collides
```

Comment:
Liam's branch (the one that has the HUD with a method you can use to update the score) is now merged. Let's get his changes brought into your branch and get this fully integrated with that HUD before we merge.

You'll have to pull my repository's main branch into your feature branch to get those updates (and resolve merge conflicts). Let me know if you need any support on that front.

Reply:
I tried my best to integrate the score addition when picking up coins but couldn't figure it out, I commented out code where it should probably go. As for the merge conflict, I resolved it in my latest commit just now but Idk why Github isn't recognizing that.